project_proposal/signatures.tex

```diff
@@ -0,0 +1,104 @@
+\begin{tabular}{@{}l l} 
+\textbf{Title:} & Bierproductie \\
+& \\
+\textbf{Institution:} & University of Southern Denmark \\
+& The Faculty of Engineering, The Mærsk Mc-Kinney Møller Institute \\
+& Campusvej 55, 5230 Odense M \\
+& \\
+\textbf{Education:} & Bachelor of Engineering in Software Technology\\
+& \\
+\textbf{Semester:} & 3. Semester \\
+& \\
+\textbf{Course Title:} & Industrial 4.0 cyber-physical software systems \\
+& \\
+\textbf{Internal Course Code:} & ST3-PRO \\
+& \\
+\textbf{Project Period:} &  31.08.2020 - 19.12.2020\\
+& \\
+\textbf{ECTS:} & 10 ECTS\\
+& \\
+\textbf{Supervisor:} & Parisa Niloofar\\
+& \\
+\textbf{Project group:} & 06\\
+& \\
+
+\\
+\end{tabular}
+
+% VARIABLES
+\newcounter{PROD}
+\setcounter{PROD} {0}
+
+%%%%
+
+
+% Jakob Jakob Jakob Jakob Jakob Jakob Jakob Jakob Jakob Jakob Jakob
+\ifnum \value{PROD}=1
+    \includegraphics[scale=0.07]{signatures/signatureJR.jpg}

```

Comment:
We are also missing signatures files, or people need to make a new one again

Reply:
I think we still have them from last term, I'll check tomorrow.

Reply:
I found the signatures and added them